.circleci/config.yml

```diff
@@ -0,0 +1,30 @@
+version: 2.1
+
+orbs:
+  python: circleci/python@1.5.0
+  browser-tools: circleci/browser-tools@1.4.0
+
+jobs:
+  test-and-lint:
+    docker:
+      - image: cimg/python:3.9
+    steps:
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - checkout
+      - python/install-packages:
+          pkg-manager: pip
+          pip-dependency-file: requirements-dev.txt
+      - run:
+          name: Lint with flake8
+          command: |
+            flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+            flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+      - run:
+          name: Run tests
```

Comment:
Before running the tests, let's fully rebuild the package. In repos like this where we're including the built package in the repo, it's too easy to let the source code and build artifacts get out of sync in a commit, and we want to test what's in the source code.